plugins/discovery-ec2/src/main/java/org/elasticsearch/cloud/aws/AwsEc2Service.java

```diff
@@ -164,20 +164,22 @@
      * Defines discovery settings for ec2. Starting with discovery.ec2.
      */
     interface DISCOVERY_EC2 {
```

Comment:
Not within this PR. May be with a more global PR which we can apply on master branch?